config/locales/client.en.yml

```diff
@@ -600,19 +597,19 @@ en:
     cancel_invitation: Cancel invitation
 
   notification_models:
-    discussion: 'thread: '
-    comment: 'comment in '
-    poll: 'poll: '
-    outcome: 'outcome: '
-    stance: 'vote: '
+    discussion: 'Thread: '
+    comment: 'Comment in '
+    poll: 'Poll: '
+    outcome: 'Outcome: '
+    stance: 'Vote: '
 
   notifications:
     header: "Notifications"
     no_notifications: "You don't have any notifications"
 
     with_title:
       comment_replied_to: "%{actor} replied to you in: %{title}"
-      discussion_announced: "%{actor} invited you to a thread: %{title}"
+      discussion_announced: "%{actor} invited you to the thread: %{title}"
```

Comment:
Sorry, I think 'a' is better here. Making a change would mean all the translations of this become invalidated.

Reply:
It's clear, but if you go after that, you shouldn't update anything in the source file! What if there really are typos? you add strings? It will show you when translations are out of date if they have been updated! And the is better precisely because you have the %{title} tag! From this I conclude that there is a single notification for each thread! Since the tag %{title} exists, it doesn't mean just any thread... As an overview:


%{actor} invited you a the thread

%{actor} invited you to the thread: %{title}

Reply:
If there are typos, I'm happy to fix them.
Yea, I guess 'the' is a little better. Thanks.